Parse/src/main/java/com/parse/ParseFileController.java

```diff
@@ -41,7 +41,7 @@ public ParseFileController(ParseHttpClient restClient, File cachePath) {
   /* package */ ParseHttpClient awsClient() {
     synchronized (lock) {
       if (awsClient == null) {
-        awsClient = ParsePlugins.get().newHttpClient();
+        awsClient = ParsePlugins.get().awsClient();
```

Comment:
Should we rename to fileClient? Files are not necessarily hosted on AWS anymore.